src/app/modules/user/user.component.styl

```diff
@@ -0,0 +1,68 @@
+.container
+  display flex
+  flex-direction column
+  width 55%
+  padding 1rem
+  margin 0 auto
+  height 100vh
+  -webkit-box-shadow 0 0 5px 1px rgba(0, 0, 0, 0.75)
```

Comment:
pretty sure you dont need the vendor prefixes for box-shadow anymore
if you are unsure you can check on caniuse